Algorithms/Hard/4_MedianofTwoSortedArrays/Solution.java

```diff
@@ -0,0 +1,40 @@
+class Solution {
+    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
+        double median = 0.0;
+        int n1 = nums1.length;
+        int n2 = nums2.length;
+        int arr[] = new int[n1 + n2];
```

Comment:
Please provide a meaningful name to `arr`, for example: `combined`, and change in other places where you've used `arr`.